tests/foreman/cli/test_activationkey.py

```diff
@@ -845,7 +845,7 @@ def test_associate_product_2(self):
 
     @run_only_on('sat')
     @stubbed()
-    def test_associate_product_3(self):
+    def test_positive_update_redhat_custom_products(self):
```

Comment:
ha ha you are absolutely right.  I meant `test_positive_update_redhat_and_custom_products`, I removed the and earlier thinking it would make sense as it is, but apparently it is not from your comments.  I will put `and` back :)